### PR TITLE
Use prettier to remove semicolons, instead of eslint

### PR DESCRIPTION
**Pull request by hawkrives**
Before: `✨  Done in 10.86s.`
After: `✨  Done in 4.16s.`

---

Prettier@1.0 supports a `--no-semi` option, to prevent printing semicolons. This allows us to quit processing every file twice, which is what we currently do: `prettier | eslint`, where we let eslint remove the semicolons for us.

### CODE REVIEW

File: source/globalize-fetch.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ function status(response) {
   if (response.status >= 200 && response.status < 300) {
     return response
   } else {
-    let error = new Error(response.statusText);
-    // attach the original response to the thrown error
+    let error = new Error(response.statusText)
+    ;// attach the original response to the thrown error
```
Comment (hawkrives) [inline source/globalize-fetch.js]: I filed an issue for this one. https://github.com/prettier/prettier/issues/1245

File: source/app.js [hunk 2]
```diff
@@ -178,15 +178,15 @@ export default class App extends React.Component {
...
+  _navigator: Navigator
 
   registerAndroidBackButton = () => {
     if (this._navigator && this._navigator.getCurrentRoutes().length > 1) {
       this._navigator.pop()
       return true
     }
     return false
-  };
+  }
```
Comment (hawkrives) [inline source/app.js:189]: ESLint didn't know how to remove semicolons from these statements.

File: source/lib/courses/index.js [hunk 3]
```diff
@@ -1,7 +1,3 @@
 // @flow
 export {loadAllCourses} from './load-all-courses'
-export type {
-  CourseType,
-  CourseCollectionType,
-  CoursesByTermType,
-} from './types'
+export type {CourseType, CourseCollectionType, CoursesByTermType} from './types'
```
Comment (hawkrives) [inline source/lib/courses/index.js:3]: The only explanation I have for these is that … the algorithm was counting the semicolon in the line length? which makes sense.

File: source/globalize-fetch.js (hunk 1 above)
Comment (drewvolz) [inline source/globalize-fetch.js]: I think you filed an issue against this.

**Pull request merged**